docs/application/tizen-studio/common-tools/certificate-registration.md

```diff
@@ -46,6 +46,9 @@ You can create a new certificate profile with the Certificate Manager:
 
 ## Managing Certificate Profiles
 
+**Note:**
+- **Please keep your certificates and password safe in local repo to prevent it from being compromised**
```

Comment:
Note tag format needs to be fixed and "Please" is not a preferred term, suggest rephrasing as: 

>**Note**
>
>It is recommended to keep your certificates and password safe in the local repo to prevent it from being compromised.